README.md

```diff
@@ -49,15 +54,15 @@ The Specifications
 If you are a developer, you may be interested in the official JSON-LD W3C
 specifications:
 
-* `JSON-LD 1.1 - A JSON-based Serialization for Linked Data`_
-* `JSON-LD 1.1 Processing Algorithms and API`_
-* `JSON-LD 1.1 Framing`_
+* [JSON-LD 1.1 - A JSON-based Serialization for Linked Data][]
+* [JSON-LD 1.1 Processing Algorithms and API][]
+* [JSON-LD 1.1 Framing][]
 
 A list of all previous specification drafts is also available.
 
 https://json-ld.org/spec/
 
-.. _list of users: https://github.com/json-ld/json-ld.org/wiki/Users-of-JSON-LD
-.. _JSON-LD 1.1 - A JSON-based Serialization for Linked Data: http://www.w3.org/TR/json-ld/
-.. _JSON-LD 1.1 Processing Algorithms and API: http://www.w3.org/TR/json-ld-api/
-.. _JSON-LD 1.1 Framing: http://www.w3.org/TR/json-ld-framing/
+[list of users]: https://github.com/json-ld/json-ld.org/wiki/Users-of-JSON-LD
+[JSON-LD 1.1 - A JSON-based Serialization for Linked Data]: http://www.w3.org/TR/json-ld/
+[JSON-LD 1.1 Processing Algorithms and API]: https://www.w3.org/TR/json-ld-api/
+[JSON-LD 1.1 Framing]: https://www.w3.org/TR/json-ld-framing/
```

Comment:
What's intended here? Human text followed by visible URL, or human text anchored with an `href`? If the former, remove the `[]` that now surrounds the human text; if the latter, remove the `: ` and wrap the URL in `(). I'm guessing the latter is the intent, so --
```suggestion
[list of users](https://github.com/json-ld/json-ld.org/wiki/Users-of-JSON-LD)
[JSON-LD 1.1 - A JSON-based Serialization for Linked Data](http://www.w3.org/TR/json-ld/)
[JSON-LD 1.1 Processing Algorithms and API](https://www.w3.org/TR/json-ld-api/)
[JSON-LD 1.1 Framing](https://www.w3.org/TR/json-ld-framing/)
```

Reply:
But if it's the former --
```suggestion
**list of users:** https://github.com/json-ld/json-ld.org/wiki/Users-of-JSON-LD
**JSON-LD 1.1 - A JSON-based Serialization for Linked Data:** http://www.w3.org/TR/json-ld/
**JSON-LD 1.1 Processing Algorithms and API:** https://www.w3.org/TR/json-ld-api/
**JSON-LD 1.1 Framing:** https://www.w3.org/TR/json-ld-framing/
```